src/test/java/touro/chess/BishopPieceTest.java

```diff
@@ -21,22 +21,28 @@ public static void beforeClass() {
     public void getMoves()
     {
         //given
-        BishopPiece bishop = mock(BishopPiece.class);
-        when(bishop.getLocation()).thenReturn(mock(Location.class));
-        when(bishop.getLocation().getRow()).thenReturn(4);
-        when(bishop.getLocation().getColumn()).thenReturn(4);
-//        doReturn(4).when(bishop.getLocation()).getRow();
-//        doReturn(4).when(bishop.getLocation()).getColumn();
+        Location location = mock(Location.class);
+        PieceColor color = PieceColor.Black;
+        BishopPiece bishop = new BishopPiece(location, color);
+        doReturn(4).when(bishop.getLocation()).getRow();
+        doReturn(4).when(bishop.getLocation()).getColumn();
         List<Move> expectedMoves = new ArrayList<>();
+        //Expected moves to the upper left
         expectedMoves.add(new Move(bishop.getLocation(), new Location(3, 3), false));
         expectedMoves.add(new Move(bishop.getLocation(), new Location(2, 2), false));
         expectedMoves.add(new Move(bishop.getLocation(), new Location(1, 1), false));
+
+        //Expected moves the lower left
         expectedMoves.add(new Move(bishop.getLocation(), new Location(5, 3), false));
         expectedMoves.add(new Move(bishop.getLocation(), new Location(6, 2), false));
```

Comment:
Might want to break the tests up into different tests not testing all of them in one test.